### PR TITLE
Make template-haskell-optics compatible with GHC 8.8.1

### PR DESCRIPTION
**Pull request by arybczak**

### CODE REVIEW

File: optics/optics.cabal [hunk 1]
```diff
@@ -146,6 +146,9 @@ benchmark folds
   hs-source-dirs:   benchmarks
   ghc-options:      -Wall -threaded
 
+  if impl(ghc >= 8.8)
+    buildable: False
+
```
Comment (adamgundry) [inline optics/optics.cabal]: What's going on here? Surely `optics` should be buildable on 8.8?

**Comment by adamgundry** [comment]
Yay, thanks for tackling this!

File: optics/optics.cabal (hunk 1 above)
Comment (arybczak) [reply optics/optics.cabal]: lens doesn't build on 8.8.1

Comment (adamgundry) [reply optics/optics.cabal]: Sorry, my bad, I missed that these were just specific components that depend on `lens`.

Comment (phadej) [reply optics/optics.cabal]: `lens` will be buildable any moment now

**Comment by arybczak** [comment]
I'm going to merge this, I created #252 for the followup.

**Pull request merged**